erpnext/payroll/doctype/salary_slip/salary_slip_dashboard.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+from frappe import _
+
+def get_data():
+	return {
+		'fieldname': 'salary_slip',
+		'transactions': [
+			{
+				'items': [_('Overtime Slip')]
+			}
+		]
+	}
```

Comment:
Move this in Document Links section in doctype as a config and remove this file

Reply:
Done